cli/utils/ipfs/get.from.ipfs.ts

```diff
@@ -21,17 +21,28 @@ export interface IpfsManifestData {
 }
 
 export default function provideGetFromIpfs(
-    ipfsHttpClient: AxiosInstance
+    ipfsHttpClient: AxiosInstance,
+    agentId: string
 ): GetFromIpfs {
     assertExists(ipfsHttpClient, 'ipfsHttpClient')
+    assertIsNonEmptyString(agentId, 'agentId')
 
     return async function getIpfsByHash(metadataHash: string) {
-        const { data } = await ipfsHttpClient.get(`/ipfs/${metadataHash}`);
+        try {
+            const { data } = await ipfsHttpClient({url: `/ipfs/${metadataHash}`, baseURL: "https://ipfs.forta.network"});
 
-        if(data && data.manifest) {
-            return data;
-        }
+            if(data && data.manifest) {
+                return data;
+            }
+
+            throw Error(`No data found for ipfs hash ${metadataHash}`)
 
-        throw Error(`No data found for ipfs hash ${metadataHash}`)
+        } catch(e) {
+            if((e.message as string).includes('status code 403')) {
```

Comment:
ideally we check the error code itself i.e. `e.response.status`

Reply:
Updated this.